python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/vlan-interfaces.j2

```diff
@@ -50,7 +50,7 @@
 {%         set row_ip_addr = vlan_interface.ip_address | arista.avd.default('-') %}
 {%         set row_ip_vaddr = vlan_interface.ip_address_virtual | arista.avd.default('-') %}
 {%         set row_varp = vlan_interface.ip_virtual_router_addresses | arista.avd.default('-') | join(", ") %}
-{%         set row_vrrp = vlan_interface.vrrp.ipv4 | arista.avd.default('-') %}
+{%         set row_vrrp = vlan_interface.vrrp_ids.ipv4.address | arista.avd.default('-') %}
```

Comment:
vrrp_ids is a list. AVD will throw error for this line.

Reply:
removed